README.md

```diff
@@ -2,6 +2,8 @@
 
 This model is used to predict the risk of West Nile Virus being present two consecutive weeks in a row. 
 
+Most of the work in this project is data wrangling to get the data into a format where each row represents the results for a week / location collection, and the interval between the dates is regular (i.e. a week apart). Once this is accomplished, a model is fit that is very similar to a generalized linear model to predict the likelihood that WNV will be present if it was present last week. This allows our teams to go out and spray for mosquitos a week earlier than they would have otherwise.
```

Comment:
@geneorama - the CLA should be in the `CONTRIBUTING.md`. When you create a pull request, GitHub automatically [links to the file](https://github.com/blog/1184-contributing-guidelines) to remind users to follow those instructions.